src/apack/zcl_abapgit_apack_helper.clas.abap

```diff
@@ -156,11 +157,17 @@ CLASS zcl_abapgit_apack_helper IMPLEMENTATION.
     LOOP AT lt_manifest_implementation INTO ls_manifest_implementation.
       CLEAR: lo_manifest_provider, lo_apack_reader.
 
-      TRY.
-          CREATE OBJECT lo_manifest_provider TYPE (ls_manifest_implementation-clsname).
-        CATCH cx_sy_create_object_error.
-          CLEAR: lo_manifest_provider.
-      ENDTRY.
+      lv_cp = cl_oo_classname_service=>get_classpool_name( ls_manifest_implementation-clsname ).
+      GENERATE REPORT lv_cp.
```

Comment:
ugh, I would like to avoid GENERATE REPORT, alternatively, check which interfaces the class implements, and skip if it implements a non-existing interface? Checking interfaces can be done via lookup in SEOsomething tables, or there is also a function module(which I think is already used somewhere in abapGit)

Reply:
There's SEOMETAREL table already used, I didn't find any SEO function module to do the same.

I need your advice about something I missed: the project "abap-platform-jak" depends on "abap-platform-yy"; "yy" may now be pulled, but not "jak" because abapGit is unable to retrieve the version from "yy" manifest class (because IF_APACK_MANIFEST prevents it from being compiled), so the next popup is displayed and prevents from pulling.

![image](https://user-images.githubusercontent.com/34005250/79687671-2bf63280-8249-11ea-9d01-7923d5b45c58.png)

What do you think about adding a button "Force pull" or "Pull anyway" so that "jak" can be pulled, and should I open a new issue or should I just adjust the title and text of the current issue?